pom.xml

```diff
@@ -450,9 +450,19 @@
                                             <skip>true</skip>
                                             <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-debezium:${camel-quarkus-test-list.version}</artifact>
                                         </test>
-                                        <test><!-- https://github.com/apache/camel-quarkus/issues/4058 -->
+                                        <test><!-- Workaround hardcoded path -->
+                                            <skip>true</skip>
+                                            <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-cxf-soap-grouped:${camel-quarkus-test-list.version}</artifact>
+                                        </test>
```

Comment:
camel-quarkus-integration-test-cxf-soap-grouped contains many useful tests that are not failing. Thus, I'd vote for skipping only those specific tests with known issues via XSLT transformation in [src/main/resources/xslt/camel/test-pom.xsl](https://github.com/quarkusio/quarkus-platform/blob/main/src/main/resources/xslt/camel/test-pom.xsl) 
The changes can be ported from 2.13 branch: [
* https://github.com/quarkusio/quarkus-platform/pull/690/files#diff-83ce56bfbc63c6c743ed24689a3e000a2a06e2857c874c85c82b9ca19149c73aR125-R142
* ](https://github.com/quarkusio/quarkus-platform/pull/690/files#diff-83ce56bfbc63c6c743ed24689a3e000a2a06e2857c874c85c82b9ca19149c73aR125-R142)

The above snippet covers also camel-quarkus-integration-test-xml where could also skip just the two failing tests and keep the rest active.

Reply:
@zbendhiba thanks for explanation. I pushed again, let me know if it's okay.

Reply:
@ppalaga this should be removed maybe in 2.13 branch https://github.com/quarkusio/quarkus-platform/blob/b13ab64a5c1830db56f682f926644f06accb621d/pom.xml#L451